src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -43,25 +43,38 @@
   
   <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <DefineConstants>FUNCTIONS_V1</DefineConstants>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
Not sure what this is... will want to look into it. I'm assuming it's related to the Compile Include sections below.

Reply:
I just read about it here: http://jameschambers.com/2017/11/duplicate-compile-items-included/
and here: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj

In short, in response to changes to the defaults of how modern .NET builds projects, this flags prevents unnecessary files from being added to our project. Since we haven't updated our .NET version significantly, I doubt this is necessary for us. I'll remove it

Reply:
Removed

Reply:
Actually, this appears to be necessary. I think the new `samples/` project is automatically getting loaded onto the FunctionsV1 project, which is causing issues because Dist. Tracing only works for FunctionsV2. Without this line, we're getting compilation errors.

I think we should kept it and add it as a ToDo to remove this once we clean-up the samples

Reply:
Added to ToDos issue: https://github.com/Azure/azure-functions-durable-extension/issues/1576